app/assets/stylesheets/components/_chart.scss

```diff
@@ -0,0 +1,18 @@
+// Include our GOV.UK components
+@import "govuk_publishing_components/all_components";
```

Comment:
I don't think we need this import here as it already happens in `application.scss`. If we're doing it to show the dependency on other components (e.g. tabs) would be good to be explicit about it (@import "govuk_publishing_components/components/tabs")

Reply:
Without this, the line 
@include govuk-visually-hidden;
blows up. 
I can switch it out for @import "../../../node_modules/govuk-frontend/all"; but I'm not sure that's any better? I don't really fully understand how the imports and css dependencies work.